helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -656,7 +660,7 @@ private RollingUpdateTaskResult rollingUpdateAwaitRunning(final DeploymentGroup
       // won't be able to undeploy the job on the next update.
       final Deployment deployment = getDeployment(host, deploymentGroup.getJobId());
       if (deployment == null) {
-        return RollingUpdateTaskResult.error("deployment for this job is very broken in ZK");
+        return RollingUpdateTaskResult.error("deployment for this job not found in zookeeper");
```

Comment:
Maybe a bit more info about how this might've happened like the message here https://github.com/spotify/helios/pull/556/files#r35044609?